server/server.js

```diff
@@ -101,6 +102,14 @@ app.get(`${apiBaseUrl}/employer-form/:id`,
     return res.json(result);
   }));
 
+app.get(`${apiBaseUrl}/coords`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health'),
```

Comment:
should be available to employers too, although I suspect you'll catch this in your next feature :)